.teamcity/src/main/kotlin/common/extensions.kt

```diff
@@ -80,6 +80,15 @@ fun Requirements.requiresNoEc2Agent() {
     doesNotContain("teamcity.agent.name", "EC2")
 }
 
+/**
+ * We have some "shared" host where a Linux build agent and a Windows build agent
+ * both run on the same bare metal. Some builds require exclusive access to the
+ * hardware resources (e.g. performance test).
+ */
+fun Requirements.requiresNotRunningOnSharedHost() {
```

Comment:
```suggestion
fun Requirements.requiresNoSharedHost() {
```

Shall we name this similar to `requiresNoEc2Agent()`?

Reply:
Sure.